tests/Doctrine/Migrations/Tests/Tools/Console/Command/MigrateCommandTest.php

```diff
@@ -422,7 +422,7 @@ public function testExecuteMigrateCancelExecutedUnavailableMigrations(): void
         $output = $this->migrateCommandTester->getDisplay(true);
 
         self::assertStringContainsString('[WARNING] You have 1 previously executed migrations in the database that are not registered migrations.', $output);
-        self::assertStringContainsString('WARNING! You are about to execute a migration in database "<unnamed>" that could result in schema changes and data loss. Are you sure you wish to continue?', $output);
+        self::assertStringContainsString('WARNING! You are about to execute a migration in database "main" that could result in schema changes and data loss. Are you sure you wish to continue?', $output);
```

Comment:
On DBAL 3.4, the default schema on SQLite is `main` which is more correct than the empty string that was reported previously.